components/EmptyListView.js

```diff
@@ -0,0 +1,38 @@
+import PropTypes from 'prop-types';
+import React, { PureComponent } from 'react';
+import { connectStyle } from '@shoutem/theme';
+import { Text, View } from '@shoutem/ui';
+import EmptyList from '../assets/images/emptyList.svg'
+
+class EmptyListView extends PureComponent {
+  static defaultProps = {
+    title: "It's empty in here",
+    message: "We couldn't find anything to show...",
+  }
+
+  render() {
+    const { message, title, style } = this.props
+
+    return (
+      <View styleName="vertical h-center">
+        <EmptyList style={style.image} />
```

Comment:
Let's use another name for this graphic. having list in it's name indicates it is a more complex list component

Reply:
Any suggestions? `EmptyStateView` is already taken, so maybe `ClearStateView`, `ZeroDataStateView`...

Reply:
EmptyListImage ? using image keyword would provide a lot of context, as we're using it as a named component